src/actions/index.js

```diff
@@ -0,0 +1,24 @@
+import { SEARCH, SET_CURRENT_FILM, SORT_BY } from './constans';
+import fetchFilms from '../Api';
+
+export const getFilms = results => ({
+  type: SEARCH,
+  results,
+});
+
+export const getFilmList = (query, typeOfQuery, dispatch) => {
+  fetchFilms(query, typeOfQuery)
+    .then((results) => {
+      dispatch(getFilms(results));
+    });
+};
+
+export const setCurrentFilm = currentFilm => ({
+  type: SET_CURRENT_FILM,
+  currentFilm,
+});
+
+export const setSortBy = sortBy => ({
```

Comment:
Could you please also use different files for different types of actions> FOr example we put films related actrions to the `movies.js` file and sort related actions to `sort.js`